backend/src/utils/post.ts

```diff
@@ -0,0 +1,6 @@
+// TODO: enhance accuracy by taking into account HTML tags (only look at text and disregard HTML)
+export const readTime = (text: string): number => {
```

Comment:
Added this simple function for now since we'll need to calculate `readTime` on the backend. To be more accurate, we'll want to count characters instead of words potentially as well as stripping out HTML tags

Reply:
Cool noice! 😄